bmo/Dockerfile.PL

```diff
@@ -20,5 +15,6 @@ RUN q{
     yum clean all
 };
 
+git_clone("git://github.com/dylanwh/bmo.git");
```

Comment:
Yup, but it's a chicken and egg problem. we can't push to master until the vendor tarballs are installed on the servers.